frontend/src/CourseCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import './CourseCard.css'; 
+import PropTypes from 'prop-types'; 
+
+const CourseCard = ({name, description}) => {
+    return <div className="CourseCard">
+                <div className="CardBody">
+                    <span>Course</span>
```

Comment:
You can add the image, I've extracted it from the design, download it from [here](shorturl.at/ltCHO)
If you don't know how to do it, check how I added a placeholder image to my component.

Reply:
Thanks for this, just added the image and it looks better indeed!